ingest/templates/datawave-deployment.yaml

```diff
@@ -131,19 +131,19 @@ spec:
         secret:
           secretName: {{ include "datawave.fullname" . }}-initconatiner-cmds
```

Comment:
```suggestion
          secretName: {{ include "datawave.fullname" . }}-initcontainer-cmds
```

Reply:
and the related line here: https://github.com/NationalSecurityAgency/datawave-helm-charts/pull/34/files#diff-03010dea64698d4f28e9f636d4539f3d938c817df3ef0221f87ae6d79a0d8781L51
I would have put this as a suggestion but Github does not support commenting on unchanged lines at the moment.
https://github.com/orgs/github/projects/4247?pane=issue&itemId=943933

Reply:
NIce catch. Pushed fix.